src/main/java/com/TheAccountant/service/impl/StripePaymentService.java

```diff
@@ -0,0 +1,177 @@
+import com.stripe.Stripe;
+import com.stripe.exception.*;
+import com.stripe.model.Charge;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.annotation.PostConstruct;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by Florin on 7/31/2017.
+ */
+@Service
+@Transactional
+public class StripePaymentService implements PaymentService {
+
+    private static final String SUCCESS_STATUS = "succeeded";
+
+    @Value("${stripe.secret.key}")
+    private String secretKey;
+
+    @Value("${payment.license.amount.in.cents}")
+    private Long paymentLicenseAmountInCents;
+
+    @Value("${payment.license.currency}")
+    private String paymentLicenseCurrency;
+
+    @Autowired
+    private PaymentStripeDao paymentStripeDao;
+
+    @Autowired
+    private PaymentDao paymentDao;
+
+    @Autowired
+    private UserUtil userUtil;
+
+    @PostConstruct
+    public void init() {
+        Stripe.apiKey = secretKey;
+    }
+
+    @Override
+    public ChargeDTO charge(ChargeDTO chargeDTO, PaymentType paymentType) throws ServiceException {
+        Map<String, Object> chargeParams = new HashMap<>();
+        if (PaymentType.USER_LICENSE.equals(paymentType)) {
+            chargeParams.put("amount", paymentLicenseAmountInCents);
+            chargeParams.put("currency", paymentLicenseCurrency);
+        } else {
+            throw new ServiceException("Payment type not recognized!");
+        }
+        chargeParams.put("description", chargeDTO.getDescription());
+        chargeParams.put("source", chargeDTO.getStripeToken());
+        Charge chargeStripe = null;
+        try {
+            chargeStripe = Charge.create(chargeParams);
+        } catch (AuthenticationException e) {
+            e.printStackTrace();
+            throw new ServiceException("Problems verifying informations about provided data! Error: " + e.getMessage());
+        } catch (InvalidRequestException e) {
+            e.printStackTrace();
+            throw new ServiceException("Invalid Payment attempt! Error: " + e.getMessage());
+        } catch (APIConnectionException e) {
+            e.printStackTrace();
+            throw new ServiceException("Invalid payment API access! Error: " + e.getMessage());
+        } catch (CardException e) {
+            e.printStackTrace();
+            throw new ServiceException("Problems validating the provided card information! Error: " + e.getMessage());
+        } catch (APIException e) {
+            e.printStackTrace();
+            throw new ServiceException("Problems accessing payment support! Error: " + e.getMessage());
+        }
+
+        // PaymentStripe - details for charging with stripe
+        PaymentStripe paymentStripe = new PaymentStripe();
```

Comment:
create PaymentUtils class and move creation of PaymentStripe object in that class